### DISCUSSION TITLE
Loading assets has a long response time

### DISCUSSION THREAD

**Discussion by jakobpevec** [directus/directus#16396] [Archive]
Hello,

I have a website on which I'm loading a few images and SVGs that are hosted as assets in directus, however, I'm getting long response times from directus. Other queries are sufficiently fast, it's just assets. Also, I'm loading pre-generated thumbnails, so it's not due to resizing. 

Is there any way I can optimize this? Assets are on the same server as directus.

Here's an example:
![image](https://user-images.githubusercontent.com/69215693/201080986-599f8850-0cae-4255-bd2d-d9fb73b9b8d7.png)
![image](https://user-images.githubusercontent.com/69215693/201081041-872e9bca-59d6-4b3c-b827-6c1b96a618ba.png)

**Comment by rock31** [comment]
Did you find a solution to this? I'm currently experiencing the same behavior.

**Comment by OSZII** [comment]
+1 Did you find a solution? Images take from 600ms up to 1.06s to load

![image](https://github.com/user-attachments/assets/fa48ece6-94ff-4ad4-8d5d-96e393b44d10)

My images are not on the server but stored in cloudflare R2 this could be an additional factor
